### ISSUE TITLE
The bad test result

### ISSUE THREAD

**Issue by HPL123** [meetshah1995/pytorch-semseg#100]
I trained the fcn8s model on cityscapes dataset and the training is wright. But the test result is very bad. I do not why and how is your test result?

**Comment by adam9500370** [comment]
Hi, @HPL123 .
Can you share your training settings (i.e., # of the epochs, batch size, image size), training logs (i.e., loss) and results (i.e., accuracy)?

**Comment by HPL123** [comment]
@adam9500370 , I used all the training settings that the source code gives. I did not modify anything and I do not why.

**Comment by Meistre** [comment]
@HPL123 
I may be in trouble over the similar problem.
I also train the fcn8s model, and the running scores(accuracy, IoU, ...) on the validation/test set largely differs from the test score with the trained model.
In short, these following scores do not match.
- validation/test score while training
- test score after training (after I trained and saved the model, use it on the test set)
The second is worse about 15~30% than first one.
